users.js

```diff
@@ -657,6 +657,7 @@ class User {
 
 		if (Tells.inbox[userid]) Tells.sendTell(userid, this);
 		SG.showNews(userid, this);
+		SG.giveDailyReward(userid, this);
```

Comment:
This isn't triggering (or some other factor is preventing it from running/working when the user connects).